src/controls/Jalousie.ts

```diff
@@ -3,6 +3,25 @@ import { Control } from '../structure-file';
 import { ControlBase, ControlType } from './control-base';
 
 export class Jalousie extends ControlBase {
+    protected positionTarget: number | undefined;
+
+    // The upDownChangeHandler callback will be triggered when any movement starts or stops.
+    // Normally we want upDownChangeHandler to clear any target as it signifies manual command
+    // has occurred. This can happen in normal use or before a target has been reached for
+    // example.
+    // But in the case of movement started as a result of an auto-position command, we don't
+    // want to clear that target.
+    // For this reason we have the below boolean that tells upDownChangeHandler to ignore the
+    // first change it sees after an auto-position command is started.
+    protected upDownAutoCommandHandled: boolean | undefined;
+    protected async upDownChangeHandler(): Promise<void> {
```

Comment:
Please move the handler below the loadAsync(). The method loadAsync() should always be the first method in the control handler classes.

Reply:
It doesn't matter, but as you wish.